kubernetes/annotations_and_labels.md

```diff
@@ -229,7 +229,7 @@ Common labels
 {{- define "labels.common" -}}
 app: {{ include "name" . | quote }}
 {{ include "labels.selector" . }}
-app.giantswarm.io/branch: {{ .Values.project.branch | quote }}
+app.giantswarm.io/branch: {{ .Values.project.branch | replace "#" "-" | replace "/" "-" | quote }}
```

Comment:
I just kept this simple since there are only 2 charts. I can also use regexReplaceAll if people think that is better.